README.md

```diff
@@ -118,7 +118,18 @@ Collection of examples that solve specific problems using small pieces of code.
 
 ## Contributing
 
-Contributions are welcome!
+- Contributions make the open source community such an amazing place to learn, inspire, and create.
+- Any contributions you make are **truly appreciated**.
+- Check out our [contribution guidelines](/CONTRIBUTING.md) for more information.
+
+<h2>
+License
+</h2>
+
+<br>
+<p>
+This project is Licensed under the <a href="./LICENSE">MIT License</a>. Please go through the License atleast once before making your contribution. </p>
+<br>
```

Comment:
Probably something like this is better

```markdown
This project is Licensed under the [MIT License](./LICENSE). Please go through the License at least once before making your contribution.
```

Reply:
![123456](https://user-images.githubusercontent.com/76593166/194460288-5be58a3e-ace2-4b2c-ab1c-140bcd71a8b7.png)

<hr>

This is what is looks like if I made that change. I think it's because LICENSE is a folder and it's searching for a file inside LICENSE where infact there is none. Hence I left as it is. Please let me know if there's a way to resolve this.